hydra-web/src/js/modules/alias.js

```diff
@@ -131,7 +131,7 @@ function(
                     "bVisible":true,
                     "bSearchable":true,
                     "mRender":function(val,type,data){
-                        return val.join(",");
+                        return "<a href='#jobs/"+encodeURIComponent(val)+"/conf'>"+val+"</a>";
```

Comment:
`val` appears to be an array (at least some times) since it's being joined in the original code. Have you been able to identify if it's an array and if it's ever more than one element?